lib/cocoapods/generator/xcconfig/aggregate_xcconfig.rb

```diff
@@ -217,8 +217,9 @@ def generate_ld_runpath_search_paths
           ld_runpath_search_paths = ['$(inherited)']
           if target.platform.symbolic_name == :osx
             ld_runpath_search_paths << "'@executable_path/../Frameworks'"
+            symbol_type = target.user_targets.map(&:symbol_type).uniq.first
```

Comment:
Is it possible that an `aggregate_target` integrate more than one `user_target`? If so what is the scenario I can use to test this out and what should the expected behavior be?